pagopa/io-app
# feat(Firma con IO): [SFEQS-1330] Update copy in QTSP clauses screen

## Short description
This PR updates copy in the QTSP clauses screen.

![Simulator Screen Shot - iPhone 13 - 2023-01-26 at 16 56 19](https://user-images.githubusercontent.com/49342144/214885099-115fb427-cf91-4204-9a09-b035ca2a6ed6.png)

## List of changes proposed in this pull request
- Update `FciQtspClausesScreen`

## How to test
Start a signing flow and on QTSP clauses screen check title and subtitle are right.
